### ISSUE TITLE
Address button art that doesn't match the recipe on this site

### ISSUE THREAD

**Issue by blackshadowshade** [buttonmen-dev/buttonmen#2036]
In #2024, it was noted that the image recipes for ABCGi and CestWhat don't match the recipes on the site.

In #2025 and #2026, it is noted that Maya and Zeno have no recipe showing on the button image.

According to a conversation on IRC, we've decided to do nothing about these mismatches at the moment, although it might be useful to add a comment to the flavour texts noting that the mismatches are intentional.

**Comment by blackshadowshade** [comment]
Linking #2114, #2116, #2118.

**Comment by blackshadowshade** [comment]
In http://www.buttonweavers.com/ui/forum.html#!threadId=554, it is noted that the button art for gman97216 indicates that the recipe should have rage instead of stinger. This also fits with the theme of the picture.

**Comment by cgolubi1** [comment]
I vote wontfix - gman97216 definitely had stinger on the DHS site, and i don't see a good reason to make a change to the recipe.  Happy to concede if others feel strongly, but IMO we shouldn't change it just to change it absent something that would be better if we did.

**Comment by dwvanstone** [comment]
I agree that we should do nothing about the recipe, but I like the idea of adding a comment to the flavour-text that the mismatch is intentional. Perhaps:  

"[Note: the dice for [Button] do not match the dice on its image. This is intentional.]"
